docs/logging-usage.rst

```diff
@@ -220,8 +220,8 @@ Create a Cloud Storage sink:
    >>> client = logging.Client()
    >>> sink = client.sink(
    ...     "robots-storage",
-   ...     filter='log:apache-access AND textPayload:robot')
-   >>> sink.storage_bucket = "my-bucket-name"
+   ...     'log:apache-access AND textPayload:robot',
+   ...     "my-bucket-name")
```

Comment:
The `destination` argument actually needs to be a URI, in one of the three forms corresponding to the three services.  In this case, it should be `storage.googleapis.com/my-bucket-name`.